### PR TITLE
Add Dockerfile and examples

### PR DESCRIPTION
**Pull request by sever-sever**
Add the Docker with all dependencies. And update README

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM debian:10
```
Comment (c-po) [inline Dockerfile]: Why not use something more recent? At least Debian 11?

Comment (sever-sever) [reply Dockerfile]: It was from the original `requirements`
```
You need a machine with at least 20 GB free space with Debian 10 (bare-metal, virtual, Docker container with --privileged flag).
```
@zdc do you know where this requirement is from?
needs testing

Comment (sever-sever) [reply Dockerfile]: Ok it works fine with Debian 11 and doesn't work with Debian 12
With debian12
```
TASK [install-grub : Mount and bind /dev /proc /sys and /mnt/wroot/boot to /mnt/inst_root] ***************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Unsupported parameters for (ansible.legacy.command) module: warn. Supported parameters include: _raw_params, _uses_shell, argv, chdir, creates, executable, removes, stdin, stdin_add_newline, strip_empty_ends."}


```

Comment (sever-sever) [reply Dockerfile]: Fix for Debian12 https://github.com/vyos/vyos-vm-images/pull/45

**Pull request merged**